bip-0352.mediawiki

```diff
@@ -337,6 +337,7 @@ If each of the checks in ''[[#scanning-silent-payment-eligible-transactions|Scan
 
 * Generate the ''input_hash'' with the smallest outpoint lexicographically, using the method described above
 * Let ''A = A<sub>1</sub> + A<sub>2</sub> + ... + A<sub>n</sub>'', where each ''A<sub>i</sub>'' is the public key of an input from the ''[[#inputs-for-shared-secret-derivation|Inputs For Shared Secret Derivation]]'' list
+** In case ''A'' is the point at infinity, the transaction is not silent payment eligible, and one should proceed with the next one.
```

Comment:
From what I understand, the point at infinity is not a valid point on the curve, so it might be better to say something like:

```suggestion
** If ''A'' is not a valid point, i.e., if the sum of the input public keys results in the point at infinity, fail
```

Reply:
Well, the point at infinity is certainly in the group (and thus a possible result of the addition). Whether it's "on the curve" depends on your definition of curve, and whether it's a "valid point" probably depends on the context. So I think the original text was slightly better (but I guess either is fine in the end).

Reply:
If "valid" is context specific, then I agree its best to mention the point at infinity specifically. "one should" feels a bit too much like a suggestion to me, perhaps something more like:

```suggestion
** If ''A'' is the point at infinity, skip the transaction
```

and then also add a bullet point to the section above, "Scanning silent payment eligible transactions"

Reply:
> and then also add a bullet point to the section above, "Scanning silent payment eligible transactions" 

nit: I don't think anyone will misinterpret this, but it's this is a tiny bit confusing because of the text just above the algorithm:
```
If each of the checks in ''[[#scanning-silent-payment-eligible-transactions|Scanning silent payment eligible transactions]]'' passes, the receiving wallet must:
```

It may be a cleaner not to add the bullet point. The checks in the bullet points seem to be rather syntactical checks, so I think it's okay to omit this one.